index.html

```diff
@@ -23,7 +23,11 @@
         <header>
           <nav>
             <a href="index.html" id="logo">My logo</a>
-            <i class="hamb bx bx-menu"></i>
+            <ul class="nav-ul">
+              <li class="nav-li"><a href="#works">portfolio</a></li>
+              <li class="nav-li"><a href="#about-me">about</a></li>
+              <li class="nav-li"><a href="#container3">contact</a></li>
+            </ul>
```

Comment:
- [ ] [OPTIONAL]: I think it would look better if you capitalized text of these links 🔗